### DISCUSSION TITLE
Now that we have the JIT can we have different colours for the borders?

### DISCUSSION THREAD

**Discussion by Martleby** [tailwindlabs/tailwindcss#4567] [Ideas]
Hi,

I could probably do this with a plugin, but wondered that now we have the JIT if we could have built in support for specifying different colours for top, right, bottom, left borders?

And if that's possible, could we also have the same for dashed, double, single etc.?

Thanks,
Martin.

**Comment by simonswiss** [comment]
Hey!

This (border color for a specific side) is actually coming to Tailwind v2.2 🎉

https://github.com/tailwindlabs/tailwindcss/pull/4404

You can already play with it by using the `@canary` version of Tailwind CSS from npm:

https://www.youtube.com/watch?v=hoFwUe97Gdc

**Reply by Martleby** [reply]
Thanks Simon - thanks also for the tip about the css triangle... I was going to ask how to do that in Tailwind :)
